README.md

```diff
@@ -65,7 +71,7 @@ Convert the following create method in Mongoose to ActiveRecord.
 How does module.exports help us with separation of concerns?
 
 ```text
-
+Modules group related code together so that all related functions are included in the same file.
 ```
```

Comment:
👍  Modules allow us to export and import certain blocks / packages of code. Anything in module.exports is available globally